docs/manuals/core/entities/data-node-mgt.md

```diff
@@ -70,6 +73,21 @@ The data nodes that are part of a **scenario**, **pipeline** or **task** can be
     task.sales_history
     ```
 
+Data nodes can be retrieved by their config_id by calling `taipy.get_entities_by_config_id()^`, which will return
```

Comment:
```suggestion
Data nodes can also be retrieved by calling `taipy.get_entities_by_config_id()^` with their config_id, which will return
```

to call a method with a parameter.

This can also be applied to the pipeline, scenario, task phrases below.